app/containers/ProductView/style.scss

```diff
@@ -0,0 +1,250 @@
+/*
```

Comment:
these media query definition....
it's somehow duplicate with what kashaf have in the other PR, let's uniform and synchronise it after the codes is merge